packages/flutter/lib/src/material/app.dart

```diff
@@ -166,6 +169,17 @@ class MaterialApp extends StatefulWidget {
   /// The colors to use for the application's widgets.
   final ThemeData theme;
 
+  /// The colors to use for the application's widgets when the host platform has
+  /// specifically requested a dark UI.
+  ///
+  /// For example, starting with Android Pie, turning on battery saver causes
+  /// apps to go into "dark mode".
+  ///
+  /// See also
```

Comment:
Be consistent with the rest of the codebase

```
/// See also:
///
/// * blah
```

This form might render to `See also - blah` in some markdown engines.

Reply:
Replaced "-" with "*"